gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -109,10 +113,15 @@ gis_password_page_save_data (GisPage *gis_page)
 
   password = gtk_entry_get_text (GTK_ENTRY (priv->password_entry));
 
-  if (strlen (password) == 0)
+  if (strlen (password) == 0) {
     act_user_set_password_mode (act_user, ACT_USER_PASSWORD_MODE_NONE);
-  else
-    act_user_set_password (act_user, password, "");
+  }
+  else {
```

Comment:
Another style nit I noticed yesterday, but didn't fix: the else should be on the same line as the closing brace:

`} else {`